lxd/openfga/model.openfga

```diff
@@ -0,0 +1,61 @@
+model
```

Comment:
Can this file have comments explaining what it is and what it defines?

Reply:
Also, in the future if we add new entity types to LXD, will this file need to be updated? 
What are the ramifications if it isn't does the new resource become inaccessible to these users or does it "fail open"?